application/features/vncpasswd.py

```
@@ -42,6 +45,7 @@ def vncpasswd(passwd_path, password=None):
 
     with open(passwd_path, "wb") as passwd:
         passwd.write(obfuscated)
+        logger.debug("Vnc Password: Write obfuscated password to {}".format(passwd_path))
```

Ruff

Use f-string instead of `format` call

Convert to f-string

(UP032)